### PR TITLE
Add toast once note was added within hidden column

### PR DESCRIPTION
**Pull request by NNKKOO**
- added a toast once note was added within hidden column
- added light- and dark-mode support
- added translations for both english and german

### CODE REVIEW

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -227,7 +227,9 @@
     "hiddenColumnsVisible": "Ausgeblendete Spalten werden dir nun angezeigt!",
     "passwordCopied": "Board geschützt und Passwort kopiert!",
     "boardMadePublic": "Das Board ist nun öffentlich!",
-    "allParticipantsDone": "Alle Teilnehmer sind fertig."
+    "allParticipantsDone": "Alle Teilnehmer sind fertig.",
+    "noteToHiddenColumn": "Du hast gerade ein Karte einer ausgeblendeten Spalte hinzugefügt",
+    "noteToHiddenColumnButton": "Spalte für alle anzeigen"
```
Comment (CronJorian) [inline public/locales/de/translation.json:232]: The other toast messages use punctuation at the end, please use them too :)

```suggestion
    "noteToHiddenColumn": "Du hast gerade ein Karte einer ausgeblendeten Spalte hinzugefügt.",
    "noteToHiddenColumnButton": "Spalte für alle anzeigen"
```

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -226,7 +226,9 @@
     "hiddenColumnsVisible": "Hidden columns are now visible to you!",
     "passwordCopied": "Board protected and password copied!",
     "boardMadePublic": "The board is now public!",
-    "allParticipantsDone": "All participants are ready."
+    "allParticipantsDone": "All participants are ready.",
+    "noteToHiddenColumn": "You just added a note to a hidden column",
```
Comment (CronJorian) [inline public/locales/en/translation.json:230]:
```suggestion
    "noteToHiddenColumn": "You just added a note to a hidden column.",
```

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -226,7 +226,9 @@
...
+    "noteToHiddenColumnButton": "reveal column"
```
Comment (CronJorian) [inline public/locales/en/translation.json:231]:
```suggestion
    "noteToHiddenColumnButton": "Reveal column"
```

File: src/components/NoteInput/NoteInput.scss [hunk 4]
```diff
@@ -67,6 +67,16 @@ $note-input__input-right: 40px;
   transition: all 0.08s ease-out;
 }
 
+.toast-button {
+  background-color: $color-backlog-blue;
+  border-style: none;
+  border-radius: 12px;
```
Comment (CronJorian) [inline src/components/NoteInput/NoteInput.scss:73]: I'd prefer to use 8px for radius, due to the SettingsButtons also have a 8px radius, so its consistent.

```suggestion
  border-radius: 8px;
```

File: src/components/NoteInput/NoteInput.scss [hunk 5]
```diff
@@ -67,6 +67,16 @@ $note-input__input-right: 40px;
...
+  height: 40px;
+  width: 80%;
+  margin: 10px;
```
Comment (CronJorian) [inline src/components/NoteInput/NoteInput.scss:76]: Please use the `$margin--` when you can.
Also I think the horizontal margin looks out of place. Maybe remove it, or center the button.

image

image

File: src/components/NoteInput/NoteInput.tsx [hunk 6]
```diff
@@ -5,19 +5,23 @@ import {Actions} from "store/action";
...
+  toggleColumnVisibility: () => void;
 }
 
-export const NoteInput = ({columnIndex, columnId, maxNoteLength}: NoteInputProps) => {
+export const NoteInput = ({columnIndex, columnId, maxNoteLength, columnIsVisible, toggleColumnVisibility}: NoteInputProps) => {
   const {t} = useTranslation();
   const dispatch = useDispatch();
   const [value, setValue] = useState("");
   const noteInputRef = useRef<HTMLInputElement | null>(null);
+  const [hiddenColumnToastHasBeenShown, setHiddenColumnToastHasBeenShown] = useState(false);
```
Comment (CronJorian) [inline src/components/NoteInput/NoteInput.tsx:24]: Thats a pretty long variable name.

**Review by CronJorian — CHANGES_REQUESTED** [review]

File: src/components/NoteInput/NoteInput.scss (hunk 5 above)
Comment (NNKKOO) [reply src/components/NoteInput/NoteInput.scss]: I removed the horizontal margin and tried different alignments and widths. I think, I would go with the 1st one.

100%width
70%width-centered
fit-content

**Review by CronJorian — APPROVED** [review]

File: src/components/NoteInput/NoteInput.scss (hunk 5 above)
Comment (CronJorian) [reply src/components/NoteInput/NoteInput.scss]: I like the second one too, and in my opinion the third one just misses some padding to the sides, but this is completely your choice which to choose :)

**Pull request merged**